docs/src/main/sphinx/release/release-413.md

```diff
@@ -0,0 +1,55 @@
+# Release 413 (12 Apr 2023)
+
+## General
+
+* Improve performance of queries involving window operations or
+  [row pattern recognition](/sql/pattern-recognition-in-window) on small
+  partitions. ({issue}`16748`)
+* Improve performance of queries with `TopN` operations. ({issue}`16753`)
+* Fix potential query failure when collecting optimizer rule execution
+  statistics. ({issue}`16960`)
+
+## Delta Lake connector
+
+* Add support for writing the `timestamp with time zone` type in
```

Comment:
Add support for nested `timestamp with time zone` values in structural data types.